metadata.php

```diff
@@ -12,15 +12,88 @@
  */
 $sMetadataVersion = '2.1';
 
+// method to toggle the visibility of the DOM elements for the module terms and conditions content
+$sToggleJs = '<script type="text/javascript">
```

Comment:
what about moving this to a separate PHP file and include it here? Would make this file more readable.

Reply:
Thanks, moved the HTML and JS blocks to different files now for better readability